about/support.html.md

```diff
@@ -67,3 +67,55 @@ Here are some things to include in your ticket:
 - UTC timestamps for when the issue occurred.
 - Prepending `LOG_LEVEL=debug` to any flyctl command will provide insight into the lower-level API calls being made.
 - If you are sending us error messages or log output, it's best to send that as plain text in the body of the email (or upload a `.txt` or `.log` file for longer outputs), rather than attached screenshots of your terminal window.
+
+## Scope of Support
+
+<div class="max-w-4xl">
+  <p class="mb-6">
+    This section highlights the products we actively work on and maintain, and therefore, the ones we are able to provide support for. While we can't offer support for user's code, you can find [links to the most common frameworks](https://fly.io/docs/getting-started/get-started-by-framework/)</a> used on Fly.io.
+  </p>
+
+  <p class="mb-8">
+    Your code is yours alone to write. We're not able to provide direct support or guidance for the specific language or framework that you use. There are many of them out there and we can't be experts in all of them, but if we do know the answer we'll always let you know.
+  </p>
```

Comment:
I feel like these two paragraphs sort of say the same thing - could we condense this down to just one? Maybe something like:
```suggestion
    This section highlights the products we actively work on and maintain, and therefore, the ones we are able to provide support for. While we can't offer direct support for your own code or the specific language or framework you use, here are some [guides to the most common frameworks](https://fly.io/docs/getting-started/get-started-by-framework/)</a> used on Fly.io.
  </p>
```